client/hwlib/debian/vendor-rust.sh

```diff
@@ -12,6 +12,17 @@ fi
 
 cargo vendor-filterer "$CARGO_VENDOR_DIR"
 
+# Delete .sh files in the vendor/vcpkg directory if it exists
+if [ -d "$CARGO_VENDOR_DIR/vcpkg" ]; then
+    find "$CARGO_VENDOR_DIR/vcpkg" -name "*.sh" -exec rm -f {} +
```

Comment:
What is the trailing plus for? Was this supposed to be \; ?

Reply:
I guess it doesn't matter much in this case. The `+` sign list files in a single line, and the `;` sign lists each new file with a new line

```bash
$ find vendor/vcpkg/ -name "*.sh" -exec ls -f {} +
vendor/vcpkg/tests/run.sh  vendor/vcpkg/tests/vcpkgrs_target.sh  vendor/vcpkg/setup_vcp.sh

$ find vendor/vcpkg/ -name "*.sh" -exec ls -f {} \;
vendor/vcpkg/tests/run.sh
vendor/vcpkg/tests/vcpkgrs_target.sh
vendor/vcpkg/setup_vcp.sh
````

However, both of the commands remove the `*.sh` files:
![image](https://github.com/canonical/hardware-api/assets/84857215/cf28b8d2-edb9-4132-be10-1c1b4b1d4f19)

Anyway, I can change it to `\;` if you want

Reply:
Nah, it's okay. I was just unaware of + mode.